### ISSUE TITLE
NVDA doesn't announce count beside each filter and also position of checkboxes

### ISSUE THREAD

**Issue by v-pprasanna** [microsoft/fluentui#21154]
### Environment Information

1. Chrome Version 96.0.4664.110 (Official Build) (64-bit)
2. Windows Version - Win10
3. Non Visual Desktop Access (NVDA) Version - 2020.3

### Describe the issue:

1. Turn on the screen reader tool.
2. Listen what the reader reads when you enter to open the dialog open.

### Please provide a reproduction of the issue in a codepen:

Providing an isolated reproduction of the issue in a codepen makes it much easier for us to help you. Here are some ways to get started:

1. [https://codepen.io/pakki-prasanna/pen/rNGvJWr]
2. Turn on the screen reader tool.
3. Listen what the reader reads when you enter to open the dialog open.

Alternatively, you can also use [https://developer.microsoft.com/en-us/fluentui#/controls/web/checkbox]

#### Actual behavior:

The narrator does not announce the name of the checkbox in the table number wise

#### Expected behavior:

The narrator should announce each checkbox of the table more descriptively such as 

Controlled Checkbox not selected 1 of 3
Controlled Checkbox not selected 2 of 3
Controlled Checkbox not selected 3 of 3
### Documentation describing expected behavior

<img width="216" alt="Checkbox_Bug" src="https://user-images.githubusercontent.com/97080970/148021810-6a90dfad-eccb-475f-b9fc-e8665a05b7a4.PNG">

**Comment by gouttierre** [comment]
@v-pprasanna - Thanks for filing this issue with us. To set expectations, the developers will review this issue once capacity allows.

@spmonahan - Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley

**Comment by tringakrasniqi** [comment]
Thank you for filing this bug @v-pprasanna. 
Seems that this is not an issue for checkboxes, because checkboxes can stand on their own and even the navigation between the checkboxes is done though tabbing. Check the example of the aria practices for checkbox (they are not announced number wise): 
https://www.w3.org/TR/wai-aria-practices-1.2/examples/checkbox/checkbox.html

Instead radio groups are the ones that screen readers must announce number wise, check the example of radios in the aria practices: 
 https://www.w3.org/TR/wai-aria-practices-1.2/examples/checkbox/checkbox.html

**Comment by smhigley** [comment]
@tringakrasniqi's comment is correct, the numbering behavior mentioned in the original issue is not expected behavior on checkboxes with NVDA (or any other screen reader I'm aware of).

If one were to put the checkboxes in a list and then navigate by list item using the NVDA list item shortcut, then indices would be announced.

I'm closing this issue since Fluent matches expected behavior, but feel free to reach out you have any other questions.
